gnovm/tests/machine_test.go

```diff
@@ -40,7 +40,7 @@ func TestMachineTestMemPackage(t *testing.T) {
 					Name: tt.name,
 					F: func(t2 *testing.T) { //nolint:thelper
 						rootDir := filepath.Join("..", "..")
-						store := TestStore(rootDir, "test", os.Stdin, os.Stdout, os.Stderr, ImportModeStdlibsOnly)
+						store := TestStore(rootDir, "test", nil, os.Stdin, os.Stdout, os.Stderr, ImportModeStdlibsOnly)
```

Comment:
create a variable that you set to nil, so we now what is this parameter.

Reply:
done in [0123512](https://github.com/gnolang/gno/pull/2932/commits/0123512cd20db2806be130955573dae78342ef0e)